src/model/userVote.ts

```diff
@@ -0,0 +1,7 @@
+enum UserVote {
+  Happy, // +1 Upvote
+  Sad, // -1 Downvote
+  None, // Did not vote
```

Comment:
what's the goal of a none vote if there's no vote ?

Reply:
just a default value